Automattic/jetpack
# Jetpack Forms: remove links from the response emails sent to site owners

As site owners reply to form responses to contact their customers, they didn't want links to wp-admin in the email. This PR removes the two links in a backwards compatible way, in case any customer has decided to modify the template already.

## Proposed changes:
* Remove the view response and edit links from the form response emails.
* Leave in the %3$s and %4$s arguments in case any site owner has changed the template already.

### Other information:


## Jetpack product discussion
pcsBup-yo-p2#comment-497
  
## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Apply PR.
Embed a contact form in a test post.
Fill in the form.
You should receive an email with the form response. It will not link to the feedback admin page, or to the post editor.

Before and after screenshots:
![Before](https://github.com/Automattic/jetpack/assets/5656673/c577872f-3b6f-45e4-b049-67d03c5499ec)
![After](https://github.com/Automattic/jetpack/assets/5656673/d291f515-ae9b-4358-872d-8103ad700d3d)
